### PR TITLE
kalai sum of element v2

### PR DESCRIPTION
**Pull request by 31673**

### CODE REVIEW

File: sum.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    } else {
+	    
+          for (index = 0; index < number; index ++) {
+	     printf("Enter the %d element:", index + 1);
+	     scanf("%d", &ptr[index]);
+ 	     sum = sum + ptr[index];
+	  } 
+	  printf("Sum of Elements:%d\n", sum);
+    } 
+    free(ptr);
```
Comment (P-Prajwal) [inline sum.c:25]: free() must be called if allocation was successful

**Comment by P-Prajwal** [comment]
Please CLOSE all older Pull request versions.

**Comment by 31673** [comment]
> Please CLOSE all older Pull request versions.

ok

File: sum.c (hunk 1 above)
Comment (31673) [reply sum.c:25]: ok

**Pull request closed without merging**